perftest/CMakeLists.txt

```diff
@@ -6,13 +6,15 @@ foreach(TEST ${PERFTESTS})
   if(FINUFFT_USE_DUCC0)
     target_compile_definitions(${TEST} PRIVATE -DFINUFFT_USE_DUCC0)
   endif()
+  target_link_libraries(${TEST} PRIVATE xsimd)
   finufft_link_test(${TEST})
```

Comment:
xsimd could be added to finufft_link_test. I can make the changes later on

Reply:
I think I know what you mean; let me try to do this myself, so I can learn :-)

Reply:
Yes, this is good.